packages/embed-react/src/hooks/use-wealthsweet-messages.ts

```
@@ -21,6 +26,16 @@ export function useWealthsweetMessages({
   onUserEvent,
   onUserIdle,
 }: UseWealthsweetMessagesProps) {
+  // Even though the message hook handles this it is nicer to fail fast now if this is the only hook the user sees
```

What is failing fast here?